catwalk/tasks/__init__.py

```diff
@@ -67,6 +68,105 @@
             id_field="id"
         )   
     ).add_metrics(QA_METRICS),
+    "mrqa::race": MrqaTask("mrqa", "race").add_instance_conversion(
+        InstanceFormat.HF_QA,
+        hfqa_conversion(
+            context_field="context",
+            question_field="question",
+            answers_field="answers",
+            id_field="id"
+        )   
```

Comment:
If these field names are always the same, can we make them defaults in `hfqa_conversion()`, and then these definitions can become much shorter?